.github/workflows/coverity.yml

```diff
@@ -0,0 +1,71 @@
+# Coverity check
+name: Coverity
+
+on:
+  push:
```

Comment:
you can remove the "debug" push trigger and we're good to go

Reply:
Done

Reply:
not done...?